src/modules/core/components/SingleUserPicker/utils.ts

```diff
@@ -7,20 +7,20 @@ export const filterUserSelection = (data, filterValue) => {
     return data;
   }
 
-  const filtered = data.filter(
-    (user) =>
+  const filtered = data.filter((user) => {
+    const { username } = user.profile;
```

Comment:
The user object might not exist ?

```suggestion
    const { username } = user?.profile;
```